Bootstrap/Default.ts

```diff
@@ -82,12 +80,18 @@ export function setupAndStart(setupString = _setupString): typeof types | null {
 
     try {
         _appInsights = require("../applicationinsights");
+        if (_appInsights.defaultClient) {
```

Comment:
Was this scenario causing issues?

Reply:
Not currently but if this function is called twice for whatever reason, the sdkVersion prefix telemetry processor would be run twice